src/view/ChewingEditor.cpp

```diff
@@ -223,8 +230,12 @@ void ChewingEditor::setupImport()
         model_, SIGNAL(importCompleted(bool, const QString&, size_t, size_t)),
         ui_.get()->notification, SLOT(notifyImportCompleted(bool, const QString&, size_t, size_t))
     );
+
+    // initialize import_count
+    import_count = 0;
```

Comment:
I wasn't so sure about why "import_count" should be static and be accumulated every time when `ChewingEditor::finishFileSelection` is invoked ?